src/members/templates/members/set/set_password.html

```diff
@@ -8,7 +8,13 @@
     <h1>Set Password Page</h1>
```

Comment:
Please rename this to Register Page, this one is a bit confusing. Could also rename the file to register.html